frontend/src/components/categories/index.tsx

```diff
@@ -172,18 +144,18 @@ export const Categories = () => {
                 </Button>
               </Grid>
             ) : null}
-            {selectedCategories.length > 0 && (
+            {deleteDialogCtl.data?.length ? (
```

Comment:
I suggest we rather show the button but use the "disabled" attribute `disabled={!deleteDialogCtl.data?.length}`